cpu/cpu_darwin.go

```diff
@@ -23,17 +23,15 @@ func getCpuInfo() (cpuInfo map[string]string, err error) {
 
 	for option, key := range cpuMap {
 		out, err := exec.Command("sysctl", "-n", option).Output()
-		if err != nil {
-			return cpuInfo, err
+		if err == nil {
+			cpuInfo[key] = strings.Trim(string(out), "\n")
 		}
-		cpuInfo[key] = strings.Trim(string(out), "\n")
 	}
 
 	mhz, err := strconv.Atoi(cpuInfo["mhz"])
```

Comment:
since we're not sure the `mhz` key exists at this point I'd check its existence first